doc/connecting.md

```diff
@@ -47,6 +47,8 @@ Note that an `impala` connection with [`impyla`](https://github.com/cloudera/imp
 %sql impala://hserverhost:port/default?kerberos_service_name=hive&auth_mechanism=GSSAPI
 ```
 
+Additionally, note that autocommit featurs for `pytds` connections will not function correctly.
```

Comment:
```suggestion
Additionally, note that autocommit feature for `pytds` connections will not function correctly.
```

Reply:
You can refer here to catch warning message
https://github.com/tonykploomber/jupysql/blob/master/src/tests/integration/test_generic_db_opeations.py#L111